### ISSUE TITLE
Ruby 2.7.x obsolete warnings

### ISSUE THREAD

**Issue by johnphamvan** [vifreefly/kimuraframework#42]
Ruby 2.7.x triggers obsolete warnings for `URI.escape`. Could we consider using the well-maintained Addressable gem (https://github.com/sporkmonger/addressable) or using a different approach like `CGI::escape` or `ERB::Util.url_encode`?

I can create a pull request if that helps.

```
/usr/local/Cellar/rbenv/1.1.2/versions/2.7.1/lib/ruby/gems/2.7.0/gems/kimurai-1.4.0/lib/kimurai/base_helper.rb:7: warning: URI.escape is obsolete
```

Thanks.

**Comment by rubyFeedback** [comment]
Yeah that was a bit annoying to deal with, but I slowly changed this in my own projects too. Most use
I actually had was via .open() aka URI.open() now.

Warnings can cause quite some extra work to projects ...
